docs/language/ql-training/cpp/program-representation-cpp.rst

```diff
@@ -66,9 +62,9 @@ Working with variables
 Working with functions
 ======================
 
-Functions are represented by the Function QL class. Each declaration or definition of a function is represented by a ``FunctionDeclarationEntry``.
+Functions are represented by the Function class. Each declaration or definition of a function is represented by a ``FunctionDeclarationEntry``.
 
-Calls to functions are modeled by QL class Call and its subclasses:
+Calls to functions are modeled by class ``Call`` and its subclasses:
```

Comment:
```suggestion
Calls to functions are modeled by the QL class ``Call`` and its subclasses:
```

Reply:
CodeQL class, I think, for the same reason as above.